core/src/main/java/org/apache/carbondata/core/datastore/chunk/reader/measure/AbstractMeasureChunkReaderV2V3Format.java

```diff
@@ -103,11 +103,14 @@ public AbstractMeasureChunkReaderV2V3Format(final BlockletInfo blockletInfo,
    * @param presentMetadataThrift
    * @return wrapper presence meta
    */
-  protected BitSet getNullBitSet(
-      org.apache.carbondata.format.PresenceMeta presentMetadataThrift) {
+  protected BitSet getNullBitSet(org.apache.carbondata.format.PresenceMeta presentMetadataThrift) {
     Compressor compressor = CompressorFactory.getInstance().getCompressor();
-    return BitSet.valueOf(
-        compressor.unCompressByte(presentMetadataThrift.getPresent_bit_stream()));
+    if (null != presentMetadataThrift.getPresent_bit_stream()) {
```

Comment:
take into a variable and check presentMetadataThrift.getPresent_bit_stream()

Reply:
fixed